ISSUES.md

```diff
@@ -0,0 +1,225 @@
+
+There are two main venues to receive support: [the forums](https://discuss.huggingface.co/) and [the github issues](https://github.com/huggingface/transformers/issues).
+
+## The Forums
+
+[The user forums](https://discuss.huggingface.co/) are supported by the wide community of the library users and backed up by developers when needed.
+
+If you have a difficulty with deploying this library or some questions, or you'd like to discuss a new feature, please first consider discussing those things at the forums. Only when you feel your subject matter has been crystalized and you still need support from the library developers do proceed to file an [Issue](https://github.com/huggingface/transformers/issues).
+
+In particular all "Please explain" questions or objectively very user-specific feature requests belong to the forums. Here are some example of such questions:
+
+* "I would like to use a BertModel within a RL-Agent for a customer support service. How can I use a BertForMaskedLM in my ChatBotModel?"
+
+* "Could you please explain why T5 has no positional embedding matrix under T5Model?"
+
+* "How should I set my generation parameters for translation?"
+
+* "How to train T5 on De->En translation?"
+
+
+## The github Issues
+
+Everything which hints at a bug should be opened as an [Issue](https://github.com/huggingface/transformers/issues).
+
+You are not required to read the following guidelines before opening an Issue. However, if you notice that your issue is not being addressed, chances are that the developers have one or several difficulties with the quality of that Issue. In which case please read the following guidelines and adjust your Issue accordingly.
+
+The guidelines:
+
+1. Before posting an Issue, first search for already posted issues, since chances are someone has already asked a similar question before you. 
+
+    If you use Google your search query should be:
+    
+    ```
+    "huggingface" "transformers" your query 
+    ```
+    
+    The first two quoted words tell Google to limit the search to the context of the Huggingface Transformers. The remainder is your query - most commonly this would be an assert line if the software fails with an assert. 
+    
+    The results of such query will typically match github Issues, huggingface forums and StackExchange and people's blogs.
+    
+    If you find relevant hints, you may choose to continue the discussion there if you have follow up questions. 
+    
+    If what you found is similar but doesn't quite answer your problem, please, post a new Issue and do include links to similar Issues or forum discussions you may have found.
+    
+2. Keep the Issue short, providing the information that you think will aid the developers to understand your situation. Put yourself in the shoes of the person who has never seen your code or knows anything about your custom setup. This mental exercise will help a lot for you to know what to share and what not.
```

Comment:
```suggestion
2. Keep the Issue short, providing the information that you think will aid the developers to understand your situation. Put yourself in the shoes of the person who has never seen your code or knows anything about your custom setup. This mental exercise will help a lot for you to guess what/what not to share.
```

Reply:
why is it bad for users to know? this is the point of such guidelines - so that they can know and not guess.

do you feel that we put too much responsibility for when we ask them to know something?

how about "This mental exercise will help to develop an intuition to what/what not to share"?

Reply:
It's just to use another word since `knows` is used in the sentence before.

Reply:
sorry, I don't see the connection - they are 2 different contexts, 2 totally different "know"

Reply:
I went with intuition, as it's even better I think.